promql/lex.go

```diff
@@ -395,6 +405,12 @@ func (l *lexer) errorf(format string, args ...interface{}) stateFn {
 func (l *lexer) nextItem() item {
 	item := <-l.items
 	l.lastPos = item.pos
+
+	// TODO(fabxc): remove for version 1.0.
```

Comment:
Yes, but that's essentially the last thing and doable within the scope of one minor (which will be the transition to 1.0). No reason for 0.19.